microsoft_entra_id/README.md

```diff
@@ -75,7 +73,7 @@ You can view these events in [Event Explorer][5].
 
 Need help? Contact [Datadog support][4].
 
-[1]: https://docs.microsoft.com/en-us/azure/active-directory/reports-monitoring/overview-reports#activity-reports
+[1]: https://docs.microsoft.com/azure/active-directory/reports-monitoring/overview-reports#activity-reports
 [2]: https://docs.datadoghq.com/logs/guide/azure-logging-guide/
 [3]: https://docs.microsoft.com/en-us/azure/active-directory/reports-monitoring/tutorial-azure-monitor-stream-logs-to-event-hub
```

Comment:
Should this be https://learn.microsoft.com/entra/identity/monitoring-health/howto-stream-logs-to-event-hub ?

Reply:
This looks like the best option, thanks!

Reply:
I think you replaced link 1 with this when I had been intending link 3. I suggested a different link for link 1, in another comment

Reply:
Thanks for clarifying, updated